utils/src/main/java/com/cloud/utils/ssh/SshHelper.java

```diff
@@ -195,6 +195,15 @@ public static void scpTo(String host, int port, String user, File pemKeyFile, St
             }
 
             String result = sbResult.toString();
+            if (result == null || (result != null && result.isEmpty())){
```

Comment:
Consider using ``StringUtils.isBlank`` which will not only make the code more expressive, but also covers the case where the string is composed only of whitespace.